docs/persistence/mq.md

```diff
@@ -58,12 +58,24 @@ We have three environments for MQ: `QA`, `TEST`, and `PROD`.
 
 === "PROD"
 
-    | QueueManager   | Hostname                 | Port |
+For production environment there is a high availability solution in place that
+offers higher availability than the standard MQ solution. Assumes that the app
+reconnects in the event of an error such as "connection broken"
+Replace the hostname with the one from the table below. 
```

Comment:
```suggestion
For production environment there is a high availability solution in place that offers higher availability than the standard MQ solution.
Assumes that the app reconnects in the event of an error such as "connection broken".
```
Vi prøver å holde en setning per linje i doc-en vår.
Fjernet også linjen om å bytte ut hostnamene, for tenker at doc-en bør fokusere på de som er nye til hver tjeneste, og da har de ingen host å bytte ut.